docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -4,18 +4,20 @@ skinparam arrowThickness 1.1
 skinparam arrowColor MODEL_COLOR
 skinparam classBackgroundColor MODEL_COLOR
 
-AddressBook *-right-> "1" UniquePersonList
-AddressBook *-right-> "1" UniqueTagList
+
+CogniCare *-right-> "1" UniquePersonList
+CogniCare *-right-> "1" UniqueTagList
 UniqueTagList -[hidden]down- UniquePersonList
 UniqueTagList -[hidden]down- UniquePersonList
 
+
 UniqueTagList -right-> "*" Tag
-UniquePersonList -right-> Person
+UniquePersonList -right-> Patient
```

Comment:
Technically we haven't change model name, so should we change name on diagram?

Reply:
Let me change it back to Person.